src/main/java/com/google/sps/data/Store.java

```diff
@@ -1,12 +1,18 @@
-public class Store {
+package com.google.sps.data;
+
+public class Store { // todo: update average rating every time a store gets a new comment
```

Comment:
Should we create an issue for this TODO?

Reply:
I'm pretty sure the function is already there. It just needs to be pulled from main branch.

Reply:
this file is not being used anywhere, i will be deleting it